src/Microsoft.VisualStudio.Web.CodeGeneration.Templating/TypeUtilities.cs

```diff
@@ -22,7 +21,6 @@ private static bool IsNonIntegerPrimitive(Type type)
                    || (type == typeof(double))
                    || (type == typeof(float))
                    || (type == typeof(Guid))
-                   || (type == typeof(string))
                    || (type == typeof(TimeSpan))
```

Comment:
This check is used to determine if its a nullable type or not. 
Without this change it would generate a controller method like 
```cs
public async Task<IActionResult> Details(string? id)
```
which would fail to compile.